datasets/oneoff/id43005d93-f59a-11ee-8591-ff101ca1cce5_1_0.cgel

```diff
@@ -1,4 +1,71 @@
+                                                        :Head (Nom
+                                                            :Mod (Nom
+                                                                :Head (N :t "Advisory"))
+                                                            :Head (N :t "Committee"))
+                                                        :Mod (PP
+                                                            :Head (P :t "on")
+                                                            :Obj (NP
+                                                                :Head (Nom
+                                                                    :Mod (Nom
+                                                                        :Head (N :t "Minority"))
+                                                                    :Head (N :t "Veterans" :l "veteran")))))
+                                                    :Supplement (Clause
+                                                        :Head (VP
+                                                            :Mod (AdvP
+                                                                :Head (Adv :p "(" :t "hereinafter"))
+                                                            :Head (VP
+                                                                :Mod (PP
+                                                                    :Head (P :t "in")
+                                                                    :Obj (NP
+                                                                        :Det (DP
+                                                                            :Head (D :t "this"))
+                                                                        :Head (Nom
+                                                                            :Head (N :t "section"))))
+                                                                :Head (VP
+                                                                    :Head (V :t "referred" :l "refer" :xpos "VBN")
+                                                                    :Particle (PP
+                                                                        :Head (P :t "to"))
+                                                                    :Comp (PP
+                                                                        :Head (P :t "as")
+                                                                        :PredComp (NP
+                                                                            :Det (DP
+                                                                                :Head (D :p "``" :t "the"))
+                                                                            :Head (Nom
+                                                                                :Head (N :t "Committee" :p "''" :p ")" :p ".")))))))))))))))))))))
```

Comment:
I see there has been normalization of quotes relative to the input string, not sure if that was intentional